weblate/templates/addons/addon_list.html

```diff
@@ -72,4 +72,51 @@
   </div>
 </div>
 
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h4 class="panel-title">{% trans "Add-on Activity Logs" %}</h4>
+  </div>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>{% trans "Timestamp" %}</th>
+        <th>{% trans "Event" %}</th>
+        <th>{% trans "Details" %}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for log in addon_activity_log %}
+        <tr>
+          <td>{{ log.created|date:"Y-m-d H:i" }}</td>
+          <td>{{ log.event }}</td>
+          <td>{{ log.details|safe }}</td>
```

Comment:
You definitely do not want to use `safe` here. In the future, some add-on specific rendering might be needed here, but for now display the JSON as is:

```suggestion
          <td>{{ log.details }}</td>
```